tools/post_proofers/ppv_report.php

```diff
@@ -186,13 +186,30 @@ function maybe_report_form_problem($message)
         }
     }
 
-    function some_sig_combo($some_id, $some_label, $sig_id, $sig_label, $final_label)
+    // Checkboxes to record level of complexity for a feature; Basic, Average, Complex
+    function feature_level_combo($feature_type)
     {
         global $action;
+        $feature_label_map = get_feature_labels(true);
+
         $problem = "";
+        $basic_id = "bsc_" . $feature_type;
+        $average_id = "avg_" . $feature_type;
+        $complex_id = "cpx_" . $feature_type;
+        $final_label = $feature_label_map[$feature_type];
         if ($action == HANDLE_ENTRY_FORM_SUBMISSION) {
-            if (isset($_POST[$some_id]) && isset($_POST[$sig_id])) {
-                $problem = _("You selected both \"Some\" and \"Significant Amount\".");
+            $numsel = 0;
```

Comment:
What you have here is clear and I'm not suggesting that you change it, but the optimizer in me kept trying to figure out if there was a simpler way to do this and I think the answer is "no, but here's a more (untested) PHP way":
```php
if (count(array_intersect([$basic_id, $average_id, $complex_id], array_keys($_POST))) > 1)
```

Reply:
Yes, I can see how that would do the same - array_intersect would return all the keys from `$_POST` that match the basic, average or complex ids for this feature type. If there's more than one match, something's gone wrong?

That code looked less unwieldy when the `if`s were single line commands, but I fell foul of php-cs-fixer :)

Reply:
I think my one-liner is probably harder to understand overall and think we should keep yours. I'll merge this in tomorrow morning my time unless you object between now and then 😁

Reply:
I'm fine with that, but thanks anyway for the teaching point